### DISCUSSION TITLE
Is it possible to have anti-afk?

### DISCUSSION THREAD

**Discussion by erho1337** [redphx/better-xcloud#148] [General]
Hello, I haven't seen it anywhere and I would like to ask about the anti-afk function - is it possible to prevent it from kicking us out after a few minutes of inactivity. Or if anyone has a way to do it, thank you in advance and best regards.

**Comment by redphx** [comment]
Duplicate #53  
It's possible, but I chose not to implement it.

**Reply by erho1337** [reply]
and I understand, and you can achieve it yourself or not, because I would care about it

**Reply by erho1337** [reply]
and how does this anti-afk work, I have to be in the game all the time or just click something because I just don't understand it
